private-templates-service/adaptivecards-templating-client/typescript/api.ts

```diff
@@ -227,8 +375,8 @@ export class Template {
             "type": "string"
         },
         {
-            "name": "isPublished",
-            "baseName": "isPublished",
+            "name": "isLive",
```

Comment:
Is there a different meaning behind "isLive" and "isPublished"?

Reply:
Whoops, isLive should be the only parameter under Template, I'll fix that!

Reply:
isLive means the same thing as isPublished, isLive is one of the fields returned by the templates after adding the different states (live, draft, deprecated)